### PR TITLE
Convert loom to h5ad. 

### PR DESCRIPTION
**Pull request by aawdeh**
This script converts a loom file into h5ad (the matrix is in csr format) -- it saves the h5ad file in the given output directory. 

It takes the loom file and the h5ad output directory as input.

### CODE REVIEW

**Review by kevinpalis — APPROVED** [review]
LGTM!

File: scripts/loom_to_h5ad.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+def convertloomtoh5ad(args):
+    """ Converts loom file format to h5ad. 
+
+    Args:
+        loom_path (str): path to loom file.  
+        output_path (str): output path directory to save h5ad file in. 
+    
+    """
+
+    file_name = os.path.basename(args.loom_file)
```
Comment (jessicaway) [inline scripts/loom_to_h5ad.py]: this can be done all at once
```suggestion
    file_name = os.path.basename(args.loom_file).split(".")[0]
```

**Review by jessicaway — APPROVED** [review]
Looks good! Very minor suggestion, up to you if you want to update

File: scripts/loom_to_h5ad.py (hunk 1 above)
Comment (aawdeh) [reply scripts/loom_to_h5ad.py]: Agreed. Thanks for the suggestion!

**Pull request merged**